frontend/packages/kubevirt-plugin/src/components/vm-disks/guest-agent-file-systems.tsx

```diff
@@ -110,9 +113,18 @@ export const FileSystemsList: React.FC<FileSystemsListProps> = ({ vmi, vmStatusB
 
   return (
     <div className="kubevirt-vm-details__file-systems">
-      <h3>File Systems</h3>
-      The following information regarding how the disks are partitioned is provided by the guest
-      agent.
+      <h3>
+        File Systems
+        <Popover
+          aria-label="File systems description"
+          position="top"
+          bodyContent={<>{GUEST_AGENT_FILE_SYSTEMS_DESCRIPTION}</>}
+        >
+          <Button variant="link" id="kubevirt-vm-details__file-systems--info-icon" isInline>
```

Comment:
is this ID needed?

Reply:
I needed it to override the PF settings that set the padding to 0px.

Reply:
Removed!